reactor-core/src/main/java/reactor/core/Exceptions.java

```diff
@@ -208,6 +208,16 @@ public static RejectedExecutionException failWithRejected() {
 	}
 
 	/**
+	 * Return a singleton {@link RejectedExecutionException} with a message indicating
+	 * the reason is due to the scheduler not being time-capable
+	 *
+	 * @return a singleton {@link RejectedExecutionException}
+	 */
+	public static RejectedExecutionException failWithNotTimeCapable() {
```

Comment:
I'd call it `failWithRejectedNotTimeCapable()`

Reply:
done